Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO && \
     make clean && \
     make
 
-RUN cp /usr/src/app-netutil/bin/samples /usr/bin/app
+RUN cp /usr/src/app-netutil/bin/go_app /usr/bin/app
```

Comment:
Shall we move the `Dockerfile` and `deployments/pod.yaml` to sample dir? for example, into `c_sample` or/and `go_sample`. This way, it will be consistent with dpdk samples. And we probably could remove `Dockerfile.rhel7` as we will not be delivering an OpenShift net-util images officially.